### PR TITLE
Added Oracle Database Professional Notes (Changes Added)

### PR DESCRIPTION
**Pull request by CodeSudo**
Added Oracle Database Professional Notes and link for pdf as well

## What does this PR do?

Add one book for oracle database notes

### Description
Oracle database book is a notes for professionals and written as by professionals
### Why is this valuable (or not)?
This book was missing in the library and will help a lot of oracle developers
### How do we know it's really free?
goalkicker.com is known. they provide multiple books for free
### For book lists, is it a book? For course lists, is it a course? etc.
Books
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -1193,14 +1193,14 @@ Books on general-purpose programming that don't focus on a specific language are
 * [AngularJS Guide](https://docs.angularjs.org/guide/) (HTML)
 * [AngularJS Material Designing](https://material.angularjs.org/latest/) (HTML)
 * [AngularJS Notes for Professionals](https://goalkicker.com/AngularJSBook) - Compiled from StackOverflow Documentation ([PDF](https://goalkicker.com/AngularJSBook/AngularJSNotesForProfessionals.pdf))
+* [Angular2 Notes for Professionals](https://goalkicker.com/Angular2Book) - Compiled from StackOverflow Documentation ([PDF](https://goalkicker.com/Angular2Book/Angular2NotesForProfessionals.pdf))
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1196]: use 1 link, the pdf link

**Comment by sohansai** [comment]
The book already exists. Please find the screenshot attached below
aaaaa

**Comment by CodeSudo** [comment]
Ok there is a copy already available! One more thing can i make some changes in commit in the same PR?

**Comment by CodeSudo** [comment]
I removed the previous pdf code and added for Oracle database notes

**Pull request closed without merging**